### PR TITLE
Update to index.html file

### PR DESCRIPTION
**Pull request by Spades0**
I made some changes

### CODE REVIEW

**Comment by Spades0** [comment]
This is a test PR

File: index.html [hunk 1]
```diff
@@ -10,7 +10,7 @@
 </head>
 <body>
     
-    <h2>make some magic here!!</h2>
+    <h2>I made magic here!!</h2>
```
Comment (Spades0) [inline index.html:13]: Let us try another change

File: index.html [hunk 2]
```diff
@@ -10,7 +10,7 @@
 </head>
 <body>
```
Comment (HaleematAdetoro) [inline index.html:11]: test

**Review by HaleematAdetoro — CHANGES_REQUESTED** [review]
test

**Review by HaleematAdetoro — APPROVED** [review]

**Pull request merged**